src/Integration.Vsix/SonarLint.vsct

```diff
@@ -39,8 +42,23 @@
           <ButtonText>SonarLint</ButtonText>
         </Strings>
       </Menu>
+      <Menu guid="guidCmdSet" id="SonarLintHelpMenuItem" priority="0x9999">
```

Comment:
We should probably be consistent between your PR and mine, as well as future commands.
Perhaps it's better to drop "Item" and "SonarLint" from the name, as we are referring here to the menu. So maybe just `HelpMenu`.
WDYT?

Same for `SonarLintHelpMenuGroup` = `HelpMenuGroup` ?